js_modules/dagster-ui/packages/ui-core/src/assets/AssetView.tsx

```diff
@@ -221,7 +222,11 @@ export const AssetView = ({assetKey, headerBreadcrumbs, writeAssetVisit, current
     if (isLoading) {
       return <AssetLoadingDefinitionState />;
     }
-    return <AssetAutomaterializePolicyPage assetKey={assetKey} definition={definition} />;
+    if (Date.now()) {
```

Comment:
Oops, will remove this in favor of turning on the new page. (Unless this is worth a feature flag? That might be excessive.)